app/uk/gov/hmrc/agentsubscriptionfrontend/service/SessionStoreService.scala

```diff
@@ -33,6 +33,12 @@ class SessionStoreService @Inject() (sessionCache: SessionCache) {
   def cacheKnownFactsResult(knownFactsResult: KnownFactsResult)(implicit hc: HeaderCarrier, ec: ExecutionContext): Future[Unit] =
     sessionCache.cache("knownFactsResult", knownFactsResult).map(_ => ())
 
+  def fetchInitialDetails(implicit hc: HeaderCarrier): Future[Option[InitialDetails]] =
```

Comment:
Please can you add tests for cache/fetchInitialDetails to `SessionStoreServiceSpec`. Hopefully this will be easy to do as the tests can be similar to the existing tests for cache/fetchInitialDetails